@dg0yt I think it is not working (not configuring with CMake). I tried commands:
```
     apt-get update && apt-get upgrade -y --no-install-recommends
     apt-get install g++ git cmake curl zip unzip tar ca-certificates make pkg-config --no-install-recommends  -y
    git clone https://github.com/dg0yt/vcpkg.git -b soci --depth=1
    ./vcpkg/bootstrap-vcpkg.sh
    ./vcpkg/vcpkg install 'soci[sqlite3]'
soci provides CMake targets:

    find_package(soci CONFIG REQUIRED)
    # Using core (loading backends at runtime)
    target_link_libraries(main PRIVATE $<IF:$<TARGET_EXISTS:SOCI::soci_core>,SOCI::soci_core,SOCI::soci_core_static>)

    # Using the sqlite3 backend directly
    target_link_libraries(main PRIVATE $<IF:$<TARGET_EXISTS:SOCI::soci_sqlite3>,SOCI::soci_sqlite3,SOCI::soci_sqlite3_static>)


    ./vcpkg/vcpkg integrate install
CMake projects should use: "-DCMAKE_TOOLCHAIN_FILE=/home/vcpkg/scripts/buildsystems/vcpkg.cmake"
   cmake . -DCMAKE_TOOLCHAIN_FILE=/home/vcpkg/scripts/buildsystems/vcpkg.cmake
```
And errors are like that:
```
cmake . -DCMAKE_TOOLCHAIN_FILE=/home/vcpkg/scripts/buildsystems/vcpkg.cmake
-- The CXX compiler identification is GNU 11.4.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at vcpkg/scripts/buildsystems/vcpkg.cmake:859 (_find_package):
  Could not find a package configuration file provided by "soci" with any of
  the following names:

    sociConfig.cmake
    soci-config.cmake

  Add the installation prefix of "soci" to CMAKE_PREFIX_PATH or set
  "soci_DIR" to a directory containing one of the above files.  If "soci"
  provides a separate development package or SDK, be sure it has been
  installed.
Call Stack (most recent call first):
  CMakeLists.txt:15 (find_package)


-- Configuring incomplete, errors occurred!
See also "/home/CMakeFiles/CMakeOutput.log".
```
my CMakeLists.txt:
```
cmake_minimum_required(VERSION 3.16)
project(Learning_VCPKG LANGUAGES CXX)

find_package(soci CONFIG REQUIRED)
# Using core (loading backends at runtime)
target_link_libraries(${PROJECT_NAME} PRIVATE $<IF:$<TARGET_EXISTS:SOCI::soci_core>,SOCI::soci_core,SOCI::soci_core_static>)

# Using the sqlite3 backend directly
target_link_libraries(${PROJECT_NAME} PRIVATE $<IF:$<TARGET_EXISTS:SOCI::soci_sqlite3>,SOCI::soci_sqlite3,SOCI::soci_sqlite3_static>)
```
C++ file the same as [here](https://github.com/microsoft/vcpkg/pull/34327#issuecomment-1756995933)

--- a/ports/soci/usage
+++ b/ports/soci/usage
@@ -1,5 +1,5 @@
 soci provides CMake targets:
 
-    find_package(soci CONFIG REQUIRED)
+    find_package(SOCI CONFIG REQUIRED)
     # Using core (loading backends at runtime)
     target_link_libraries(main PRIVATE $<IF:$<TARGET_EXISTS:SOCI::soci_core>,SOCI::soci_core,SOCI::soci_core_static>)

--- a/versions/s-/soci.json
+++ b/versions/s-/soci.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "9811b837f5656b6238b497a0fa2ba162243ee1a2",
+      "git-tree": "24c340284022dab38da690548caa65c4f003bb7c",
       "version": "4.0.3",
       "port-version": 3
     },

